include/swift/AST/TypeLoc.h

```diff
@@ -29,7 +29,7 @@ class TypeRepr;
 
 /// TypeLoc - Provides source location information for a parsed type.
 /// A TypeLoc is stored in AST nodes which use an explicitly written type.
-class alignas(1 << TypeLocAlignInBits) TypeLoc {
+class alignas(8) TypeLoc final {
```

Comment:
Could you please keep it as `1 << TypeReprAlignInBits` until `TypeLoc` gets removed?

Reply:
Ah, sure thing!